### ISSUE TITLE
Washed theme colors

### ISSUE THREAD

**Issue by ArmynC** [amir1376/ab-download-manager#227]
Although I may be mistaken about this one, the color scheme of the theme is hard and rather tiring on eyes. By swaping it on Light mode, it is even worse. The white color is a diminished white-grayish spectrum.

For starters, you may check [Coolors](https://coolors.co/) and even [ColorHunt](https://colorhunt.co/palettes/popular). Thanks!

**Comment by amir1376** [comment]
Hi there thanks for your guidance. I also agree that it should be improved

**Comment by swanknight** [comment]
I agree, I hope you add a "Custom" theme option to set the background/fore color manually.
